src/components/ChatModeSelector.tsx

```diff
@@ -53,7 +55,14 @@ export function ChatModeSelector() {
             <SelectValue>{getModeDisplayName(selectedMode)}</SelectValue>
           </MiniSelectTrigger>
         </TooltipTrigger>
-        <TooltipContent>Open mode menu</TooltipContent>
+        <TooltipContent>
+          <div className="flex flex-col">
+            <span>Open mode menu</span>
+            <span className="text-xs text-muted-foreground">
```

Comment:
I think this color needs to be adjusted b/c it's not legible in light mode:

<img width="333" height="157" alt="Image" src="https://github.com/user-attachments/assets/7bc4c569-37e4-4c25-bcd9-b3774aa64a55" />

Reply:
thanks, looks good!